.github/workflows/pythontesting.yml

```
@@ -47,7 +52,7 @@ jobs:
       shell: bash -l {0}
       run: |
         conda activate ramp-ua
-        pytest --cov=microsim tests/ --cov-report=xml
+        pytest --full-trace --cov=microsim tests/ --cov-report=xml
```

This needs switching back
```suggestion
        pytest --cov=microsim tests/ --cov-report=xml
```